css/index.css

```diff
@@ -144,7 +320,7 @@ background-color: #4a8aff;
     background-color: #25a7b0;
     color: white !important;
 }
-.feedback{
-    cursor: pointer;
-    font-size: 185%;
-}
\ No newline at end of file
+/*.feedback{*/
```

Comment:
Any reason to leave it commented? It can always be retrieved by going back to previous commits.